### DISCUSSION TITLE
Some of the taskbar items like Copilot and Widgets are never visible in Windows 10 taskbar

### DISCUSSION THREAD

**Discussion by VarunAgw** [valinet/ExplorerPatcher#2928] [Ideas]
The following items are never visible when using Windows 10 taskbar with Explorer patcher.

![image](https://github.com/valinet/ExplorerPatcher/assets/4201835/da031bcf-07f9-487a-bfeb-15a348d80150)

**Comment by pyrates999** [comment]
Copilot probably isn't included in the windows 10 taskbar code that windows 11 has because Microsoft never thought you'd be running the windows 10 taskbar in windows 11.

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by VarunAgw** [comment]
Edition	Windows 11 Pro
Version	23H2
Installed on	‎21/‎11/‎2023
OS build	22631.3155
Experience	Windows Feature Experience Pack 1000.22684.1000.0


Explorer Patcher: 22621.3007.63.4

I am using default settings for Explorer Patcher

I use 7+TT but I exited it for testing.

I should add this issue is happening for years now.

**Reply by pyrates999** [reply]
It makes sense Microsoft would only add copilot to the windows 11 taskbar, and the other applications that you pointed out, and not the windows 10 taskbar that is in windows 11.

**Comment by milestprower92** [comment]
> The following items are never visible when using Windows 10 taskbar with Explorer patcher.
> 
> ![image](https://private-user-images.githubusercontent.com/4201835/309252238-da031bcf-07f9-487a-bfeb-15a348d80150.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk0MDUzMzksIm5iZiI6MTcwOTQwNTAzOSwicGF0aCI6Ii80MjAxODM1LzMwOTI1MjIzOC1kYTAzMWJjZi0wN2Y5LTQ4N2EtYmZlYi0xNWEzNDhkODAxNTAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDJUMTg0MzU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGFiODhmYjU3YjQyNTIwMzc2YWFmNGRjMDY2NzJhYjc2M2Y2OGFkOTNiYzY5MjFlNWU3OGZjMGY4MDM3ZmYxNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.RuUGL713xJOOULjmOJvOPGrqAAtVKm0ODDZ8ncWpjTA)

those items doesn't show up on the windows 10 taskbar because is only available on the windows 11 taskbar

**Comment by zake2002v2** [comment]
There's been a recent update in Windows 10 that brings back the copilot button in the Windows 10 taskbar, but the explorer patcher doesn't currently support it in itself yet, in Windows 10 the copilot button is [located where the notification button should be](https://github.com/valinet/ExplorerPatcher/assets/42307974/41a6f42b-03ff-4965-bab9-4815f4d6f8c2), or rather sitting between that and the show desktop button at the edge of the taskbar.

I still wish to suggest that it can instead replace the no longer working Cortana button [like I asked in another discussion forum here](https://github.com/valinet/ExplorerPatcher/discussions/2501#discussioncomment-8524734), probably as another option in the explorer patcher configs
[Actually, on second thought, I'll just make that into its own separate idea](https://github.com/valinet/ExplorerPatcher/discussions/2948)
